views/js/pciCreator/dev/audioRecordingInteraction/runtime/audioRecordingInteraction.js

```diff
@@ -336,28 +342,25 @@ define([
             this._cleanDelayCallback();
 
             // waiting for permission
-            this.askPermissionAccessMic(function() {
-
+            this.askPermissionAccessMic(function () {
                 self.countdown.start();
 
                 // adding a delay before start recording...
-                self._delayCallback = setTimeout(function() {
-
+                self._delayCallback = setTimeout(function () {
                     self.countdown.destroy();
 
                     // restore controls states
-                    _.forEach(self.controls, function(ctr, id) {
+                    _.forEach(self.controls, function (ctr, id) {
                         ctr.setState(self.ctrCache[id]);
                     });
 
                     self._cleanDelayCallback();
 
-                    if (!self.hasMediaStimulus() || self.hasMediaStimulus() && self.mediaStimulusHasPlayed()) {
+                    if (!self.hasMediaStimulus() || (self.hasMediaStimulus() && self.mediaStimulusHasPlayed())) {
```

Comment:
Condition is changed

Reply:
It would work in the same manner, && ll be evaluated first and then the || ll be evaluated because of precedence: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_Precedence